File: 2011/main.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	fmt.Println(finalValueAfterOperations([]string{"--X","X++","X++"}))
}

func finalValueAfterOperations(operations []string) int {
	x:=0
	for i := range operations {
		if strings.Contains(operations[i],"++"){
			x++
		}else if strings.Contains(operations[i],"--"){
			x--
		}else{
			continue
		}
	}
	return x
}

File: 2011/main.py
class Solution(object):
    def finalValueAfterOperations(self, operations):
        x = 0
        for operation in operations:
            if "++" in operation:
                x += 1
            elif "--" in operation:
                x -= 1
        return x
solution = Solution()
result = solution.finalValueAfterOperations(["--X", "X++", "X++"])
print(result)